omnigibson/simulator.py

```diff
@@ -461,6 +515,57 @@ def viewer_width(self, width):
             """
             self._viewer_camera.image_width = width
 
+        def add_ground_plane(self, floor_plane_visible=True, floor_plane_color=None):
+            """
+            Generate a ground plane into the simulator.
+            """
+            if self._floor_plane is not None:
+                return
+            plane = lazy.omni.isaac.core.objects.ground_plane.GroundPlane(
+                prim_path="/World/ground_plane",
```

Comment:
Should this prim path be the same prim path as `self._floor_plane` or is this difference intentional?

Reply:
TODO: add a new variable to capture /ground_plane